srai/utils/constants.py

```diff
@@ -1,3 +1,6 @@
 """Constants used across the project."""
 
 WGS84_CRS = "EPSG:4326"
+
+GTFS2VEC_TRIPS_PREFIX = "trip_count_at_"
+GTFS2VEC_DIRECTIONS_PREFIX = "directions_at_"
```

Comment:
I think these should be placed somewhere else. They are related to Gtfs2Vec specifically, rather than the whole library, right?
We should take care not to make this place a garbage dump for all constants

Reply:
I will move them to loader file then